lib/ansible/plugins/inventory/docker_swarm.py

```diff
@@ -56,6 +56,13 @@
         tls_hostname:
             description: When verifying the authenticity of the Docker Host server, provide the expected name of the server.
             type: str
+        host_uri:
```

Comment:
The name `host_uri` suggests that this option *expects* a host URI, not that it provides one. How about:
```suggestion
        add_host_uri:
```
Or `insert_host_uri`, `inject_host_uri`, `provide_host_uri`, ...?

Reply:
I don't mind any name :) @morph027 any preference?